documentation/preview-push.md

```diff
@@ -92,9 +92,9 @@ OPTIONS:
     generally used for reproducibility in testing pipelines.
 
 * `--report name`
-  * (`push` only!)  Generate a machine-parseable report of
-    performed corrections in the file named `name`. If no name is specified, no
-    report is generated.
+  * Write a machine-parseable report of
+    corrections to the file named `name`. If no name is specified, no
+    report is generated. See [JSON Reports](../advanced-features/json-reports.md)
```

Comment:
Relative paths should be used:

```suggestion
    report is generated. See [JSON Reports](json-reports.md)
```

Reply:
I don't understand how relative URLs work in markdown.  Or does our automation inserts the correct relative path?

Reply:
These are not relative URLs, but relative file paths (on disk). Most IDEs will provide an autocompletion/clickable link for this so you can move on to the Markdown file.

![Screenshot 2024-10-09 at 22 15 51](https://github.com/user-attachments/assets/1fb11e60-8fa2-4c20-849f-779c9308a889)